integrations/haproxy/haproxy-monitoring/README.md

```diff
@@ -0,0 +1,153 @@
+
+This integration provides the following resources:
+
+* `haproxy-metrics` [check]
+* `sensu/haproxy-check:0.1.2` [asset]
+
+### Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+![](img/haproxy-overview.png)
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Add the "haproxy" subscription to agents that should run this check.
+
+   **NOTE:** set the optional `haproxy_url` agent annotation to override the default URL (`unix:///run/haproxy/admin.sock`)
+
+2. More configuration options are available for this plugin; please see the related plugin documentation for more details.
+
+## Plugins
+
+<!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
+
+- [sensu/haproxy-check:0.1.2][sensu-haproxy-check-bonsai] ([GitHub][sensu-haproxy-check-github])
+
+## Metrics & Events
+
+<!-- List of all metrics or events collected by this integration. -->
+
+This integration collects the following [metrics]:
+
+* `haproxy_active_servers`
+  number of servers active
+
+* `haproxy_backup_servers`
+  number of backup servers
+
+* `haproxy_bin`
+  bytes in
+
+* `haproxy_bout`
+  bytes out
+
+* `haproxy_chkfail`
+  healthcheck failed
+
+* `haproxy_ctime`
+  haproxy_ctime connect time
+
+* `haproxy_dreq`
+  requests denied (security)
+
+* `haproxy_dresp`
+  response denied (security)
+
+* `haproxy_econ`
+  number of connection errors
+
+* `haproxy_ereq`
+  number of request errors
+
+* `haproxy_eresp`
+  number of response errors
+
+* `haproxy_http_response_1xx`
+  response status 1xx (info)
+
+* `haproxy_http_response_2xx`
+  response status 2xx (success)
+
+* `haproxy_http_response_3xx`
+  response status 3xx (redirect)
+
+* `haproxy_http_response_4xx`
+  response status 4xx (failed)
+
+* `haproxy_http_response_5xx`
+  response status 5xx (internal error)
+
+* `haproxy_http_response_other`
+  response status (other)
+
+* `haproxy_qcur`
+  current queued requests
+
+* haproxy_qmax 
+  `max queued requests`
+
+* `haproxy_qtime`
+  queue time
+
+* `haproxy_rate`
+  session rate
+
+* `haproxy_rtime`
+  response time
+
+* `haproxy_scur`
+  session current
+
+* `haproxy_slim`
+  session limit
+
+* `haproxy_smax`
+  session max
+
+* `haproxy_ttime`
+  average time
+
+* `haproxy_weight`
+  weight
+
+* `haproxy_wredis`
+  warning redispatched
+
+* `haproxy_wretr`
+  warning retries
+
+## Alerts
+
+<!-- List of all alerts generated by this integration. -->
+
+Currently, this integration only produces alerts when metrics collection fails.
+
+## Reference Documentation
+
+<!-- Please provide links to any relevant reference documentation to help users learn more and/or troubleshoot this integration. -->
+
```

Comment:
1.  HAProxy management guide: [Statiistics and monitoring](https://cbonte.github.io/haproxy-dconv/2.4/management.html#9)